### PR TITLE
Deploy to heroku

### PR DESCRIPTION
**Pull request by abbyjoydemarest**
pipenv install!!

-this is now set up to deploy to heroku...
-you were both added to the project on heroku... support-me

### CODE REVIEW

**Comment by juanpola** [comment]
looks great!

File: support_me/settings.py [hunk 1]
```diff
@@ -148,3 +158,6 @@
 LOGIN_REDIRECT_URL = '//'
 
 SITE_ID = 2
+
+#heroku
+django_heroku.settings(locals())
```
Comment (yyapuncich) [inline support_me/settings.py]: Should we add debug = FALSE if on heroku?
I've only done that by making heroku settings another file... is there a way to do it directly in this settings file?

**Review by yyapuncich — CHANGES_REQUESTED** [review]
Question about debug = False in settings.

File: support_me/settings.py (hunk 1 above)
Comment (abbyjoydemarest) [reply support_me/settings.py]: I added heroku_settings.py. Good catch!

Comment (yyapuncich) [inline support_me/settings.py]: we can delete this line 163

File: support_me/settings.py [hunk 2]
```diff
@@ -11,10 +11,19 @@
 """
 
 import os
+import django_heroku
```
Comment (yyapuncich) [inline support_me/settings.py]: can delete line 14

**Review by yyapuncich — CHANGES_REQUESTED** [review]

**Review by yyapuncich — APPROVED** [review]

**Pull request merged**